packages/main/test/pages/List_test_page.html

```diff
@@ -322,6 +322,13 @@ <h2 id="testHeader">Test aria</h2>
 		</ui5-li-custom>
 	</ui5-list>
 
+	<ui5-list id="highlight">
```

Comment:
As we add the feature to ListItemBase, could you also test with the rest of the items to make sure it works for all of them, not only for the StandardListItem. 
I guess you already checked this, but ListItemBase is super class for a lot of classes, make sure "highlight" feature is relevant for all of them - group list item, custom list item, standard list item (is ok), NotificationListItem,  NotificationLisGtoupItem (mostly doubt about these)

Reply:
Hello @ilhan007,

Thanks for you comment.
We will stick to the initial implementation and will keep the feature to the ListItem.
I double checked and as you have mentioned, the "highlight" feature is not relevant for the classes, which extend ListItemsBase (NotificationListItem, NotificationLisGtoupItem) - in UI5 those classes do inherit the property from ListItemBase, but have no visual representation and do not have it implemented in their visual spec as well.

Reply:
After the change, the number of classes to inherit the property is: 

<img width="307" alt="Screenshot 2024-03-14 at 7 55 25" src="https://github.com/SAP/ui5-webcomponents/assets/15702139/ca6ef3f8-8f62-4085-b2eb-4affc99bd367">

I wanted to confirm if that's fine for these  (UploadCollectionItem, TreeItem, TreeItemCustom ) to get it as well.
Thanks.

Reply:
Yes, all of the items, which extend the ListItem do get the new feature.